HopsanGUI/GUIObjects/AnimatedComponent.cpp

```diff
@@ -393,7 +393,7 @@ void AnimatedComponent::updateAnimation()
 
     if(mpModelObject->getTypeName() == HOPSANGUISCOPECOMPONENTTYPENAME)
     {
-        LogDataHandler2 *pHandler = mpModelObject->getParentContainerObject()->getLogDataHandler();
+        LogDataHandler2* pHandler = mpModelObject->getParentContainerObject()->getLogDataHandler().data();
```

Comment:
Not sure if this must be fixed now, as it is a lot of work. But in all these places you add .data() to convert back to a raw pointer. It will work as before, but it would be better to rewrite the code to make use of the shared pointer (or maybe a weak pointer it Qt has such a thing). This feels like cheating.... well maybe somthing for some later time......